src/Behat/Gherkin/Parser.php

```diff
@@ -482,7 +482,7 @@ protected function parseOutline()
             }
         }
 
-        if (empty($examples)) {
+        if (!count($examples)) {
```

Comment:
I prefer to use `if (count($examples) === 0) {`, I think it is easier to read and clearer to understand. Just an opinion, feel free to ignore it

Reply:
I don't have a strong opinion against it. I'll replace it later.

Reply:
done